parts/linux/cloud-init/artifacts/mariner/cse_install_mariner.sh

```diff
@@ -3,21 +3,25 @@
 echo "Sourcing cse_install_distro.sh for Mariner"
 
 removeContainerd() {
-    retrycmd_if_failure 10 5 60 dnf remove -y moby-containerd
+    containerdPackageName="containerd"
+    if [[ $OS_VERSION == "2.0" ]]; then
+        containerdPackageName="moby-containerd"
+    fi
+    retrycmd_if_failure 10 5 60 dnf remove -y $containerdPackageName
 }
 
 installDeps() {
     dnf_makecache || exit $ERR_APT_UPDATE_TIMEOUT
     dnf_update || exit $ERR_APT_DIST_UPGRADE_TIMEOUT
-    for dnf_package in blobfuse ca-certificates check-restart cifs-utils cloud-init-azure-kvp conntrack-tools cracklib dnf-automatic ebtables ethtool fuse git inotify-tools iotop iproute ipset iptables jq kernel-devel logrotate lsof nmap-ncat nfs-utils pam pigz psmisc rsyslog socat sysstat traceroute util-linux xz zip; do
+    for dnf_package in ca-certificates check-restart cifs-utils cloud-init-azure-kvp conntrack-tools cracklib dnf-automatic ebtables ethtool fuse git inotify-tools iotop iproute ipset iptables jq kernel-devel logrotate lsof nmap-ncat nfs-utils pam pigz psmisc rsyslog socat sysstat traceroute util-linux xz zip blobfuse2 nftables iscsi-initiator-utils; do
       if ! dnf_install 30 1 600 $dnf_package; then
         exit $ERR_APT_INSTALL_TIMEOUT
       fi
     done
 
     # install additional apparmor deps for 2.0;
     if [[ $OS_VERSION == "2.0" ]]; then
-      for dnf_package in apparmor-parser libapparmor blobfuse2 nftables iscsi-initiator-utils; do
+      for dnf_package in apparmor-parser libapparmor blobfuse; do
```

Comment:
Not directly related to your change but clean up the comment above, this 2.0 specific install isnt just apparmor deps.

Reply:
Cleaned up the comment above